lib/kernel.js

```diff
@@ -101,7 +101,10 @@ module.exports = function(config, cb) {
 
   function stop() {
```

Comment:
two questions: 
are they different to the ones you mentioned here? https://github.com/nearform/nscale-kernel/blob/master/lib/kernel.js#L131 

What are the services?